lib/Parser.js

```diff
@@ -638,11 +641,14 @@ Parser.prototype.walkClassDeclaration = function walkClassDeclaration(statement)
 };
 
 Parser.prototype.walkSwitchCases = function walkSwitchCases(switchCases) {
-	switchCases.forEach(function(switchCase) {
-		if(switchCase.test)
+	for(var index = 0; index <= switchCases.length - 1; index++) {
```

Comment:
This is the more common loop style:

`for(var index = 0; index < switchCases.length; index++) {`